features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx

```diff
@@ -543,12 +538,35 @@ class InfoBar extends React.Component {
      */
     toggleOverview = (todo) => {
         if (typeof todo === 'boolean') {
-            this.setState(state => ({ showOverview: todo }));
+            this.setState({ showOverview: todo });
         } else {
             this.setState(state => ({ showOverview: !state.showOverview }));
         }
     };
 
+    /**
+     * [Temporary function] to get the first hybrid https or http endpoint of an API
+     *
+     * @param {Api} api API object
+     * @returns {string}
+     */
+    getHttpsEP = (api) => {
+        const epUrls = api.endpointURLs.filter((v) => {
+            if (v.environmentType === 'hybrid') {
+                return v;
+            } else {
+                return null;
```

Comment:
I think we don't need to return null here, We could simplify this as.

```javascript
api.endpointURLs.filter( url => environmentType === 'hybrid' )
```

Reply:
Fixed!